components/formats-bsd/src/loci/formats/FileStitcher.java

```diff
@@ -668,6 +668,7 @@ public void setOriginalMetadataPopulated(boolean populate) {
     for (ExternalSeries s : externals) {
       String[] f = s.getFiles();
       for (String file : f) {
+        file = new Location(file).getAbsolutePath();
```

Comment:
New file variable shadows the existing file variable; consider using a different name?  Also repeated in the next hunk.